rssfeed/templatetags/rssfeed_tags.py

```diff
@@ -7,5 +7,5 @@
 
 @register.inclusion_tag("rssfeed/inclusion_tags/render_rssfeed.html")
 def render_rssfeed(count=5):
-    entries = Entry.objects.all()[:count]
+    entries = Entry.objects.all()[:count].prefetch_related("feed")
```

Comment:
You're absolutely right, select_related does only one query, whereas prefetch does 2.